openedx/core/djangoapps/user_api/accounts/image_helpers.py

```diff
@@ -37,32 +37,32 @@ def _make_profile_image_name(username):
     return hashlib.md5(settings.PROFILE_IMAGE_SECRET_KEY + username).hexdigest()
 
 
-def _get_profile_image_filename(name, size):
+def _get_profile_image_filename(name, size, version):
     """
     Returns the full filename for a profile image, given the name and size.
     """
-    return '{name}_{size}.jpg'.format(name=name, size=size)
+    return '{name}_{size}.{version}.jpg'.format(name=name, size=size, version=version)
 
 
-def _get_profile_image_urls(name):
+def _get_profile_image_urls(name, version):
     """
     Returns a dict containing the urls for a complete set of profile images,
     keyed by "friendly" name (e.g. "full", "large", "medium", "small").
     """
     storage = get_profile_image_storage()
     return {
-        size_display_name: storage.url(_get_profile_image_filename(name, size))
+        size_display_name: storage.url(_get_profile_image_filename(name, size, version))
```

Comment:
it would be possible to find for any/all images for a given user, detect older unused set of images that were orphaned for whatever reason, etc, based on matching the beginning of filenames.  agreed, though, we need to figure out exactly how this is going to work.  i will follow up with a proposal.